app/views/abuse_reports/new.html.erb

```diff
@@ -8,49 +8,77 @@
 
 <!--main content-->
 <div class="userstuff">
-  <p><%= t(".purview.about",
-            tos_link: (link_to t(".purview.tos"), tos_path),
-            support_link: (link_to t(".purview.support"), new_feedback_report_path)).html_safe %></p>
-  <p><%= t(".do_not_spam_html") %></p>
-  <p>
-    <%= t(".reportable.intro") %>
+  <p class="important"><%= t(".purview.about",
+              tos_link: (link_to t(".purview.tos"), tos_path, name: "reportwhat")).html_safe %>
   </p>
+
+  <p><%= t(".purview.technical_help",
+              support_link: (link_to t(".purview.support"), new_feedback_report_path)).html_safe %>
+  </p>
+
+  <p><%= t(".purview.dmca_takedown",
+              dmca_link: (link_to t(".purview.dmca"), dmca_path)).html_safe %>
+  </p>
+
+  <p><strong><%= 
+              link_to "", name: "reportwhy" %>
```

Comment:
It looks like Rails is rendering this as
```html
<a href="/abuse_reports/new?name=reportwhy"></a>
```
instead of
```html
<a name="reportwhy"></a>
```